tee_launcher/pytest.ini

```diff
@@ -0,0 +1,6 @@
+[pytest]
+markers =
+    slow: marks tests as slow (deselect with '-m "not slow"')
+    ci_excluded: marks tests that should be excluded from ci
+    no_atexit_cleanup: Skip the autouse atexit cleanup fixture for this test
```

Comment:
from this list we are only using ci_excluded. There is no need to add the others